--- Twisted-17.1.0/src/twisted/python/_setup.py.omv~	2017-04-23 18:42:07.508400361 +0200
+++ Twisted-17.1.0/src/twisted/python/_setup.py	2017-04-23 18:42:33.012655703 +0200
@@ -193,7 +193,8 @@ _EXTENSIONS = [
         "twisted.runner.portmap",
         sources=["src/twisted/runner/portmap.c"],
         condition=lambda builder: not _PY3 and
-                                  builder._check_header("rpc/rpc.h")),
+                                  builder._check_header("rpc/rpc.h"),
+        libraries=["tirpc"]),
     ]
 
 
